contributors/design-proposals/bootstrap-discovery.md

```diff
@@ -70,16 +70,16 @@ Note that TLS bootstrap (which establishes a way for a client to authenticate it
 If the ClusterInfo information is hosted in a trusted place via HTTPS you can just request it that way.  This will use the root certificates that are installed on the system.  It may or may not be appropriate based on the user's constraints.
```

Comment:
You haven't defined ClusterInfo. I think you just mean the KubeConfig file with the list of addresses to API Servers with the `cluster-info` ConfigMap exposed and the CA cert(s)